lighthouse-core/scripts/gcp-collection/extract-data.sh

```diff
@@ -3,21 +3,32 @@
 set -euxo pipefail
 
 ##
-# @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+# @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
 # Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 # Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 ##
 
+# Unarchives "lhr-data-*.tar.gz" files inside gcp-data folder, places
+# them inside gcp-data/data.
+
 DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 LH_ROOT="$DIRNAME/../../.."
 GCP_DATA="$DIRNAME/gcp-data"
-cd $GCP_DATA
-touch extract_failures.log
+OUTPUT_DIR="$GCP_DATA/data"
+mkdir -p "$OUTPUT_DIR"
+
+cd "$GCP_DATA"
+
+rm -rf extract_failures.log
 
+# Change this line if you want to extract trace data instead.
+i=0
 for f in lhr-data-*.tar.gz; do
   echo "Extracting $f...\n"
-  tar -xzvf $f  data || echo "Failed to extract $f\n" >> extract_failures.log
+  tar -xzvf $f data || echo "Failed to extract $f\n" >> extract_failures.log
+  mv "$OUTPUT_DIR/urls.txt" "$OUTPUT_DIR/urls-$i.txt"
+  ((i += 1))
 done
 
-cd $DIRNAME
-node analyze-lhr-data.js "$GCP_DATA/data" $1 > "$LH_ROOT/.tmp/analyze-results.json"
\ No newline at end of file
+echo "Run to analyze data:"
+echo   node "$DIRNAME/analyze-lhr-data.js" "$OUTPUT_DIR" 'audit-id' > "$LH_ROOT/.tmp/analyze-results.json"
```

Comment:
I don't quite understand the motivation for this change. 

if it's about being able to rerun it easily, how about run + print or a separate script?

if it's about not knowing which audit id to run, check at the beginning?

the hardcoded `audit-id` looks like a mistake right now, `${1:-audit_id_goes_here}` maybe?

Reply:
I needed to extract, but had my own analysis script. The one provided is generic. Instead of stuffing the generic command in a README I kept it here via a print to somewhat maintain the same workflow.

This is small incremental improvement, really a more robust analysis script is needed here, but we've only ever used this thing twice so seems to early to know what it should be.

Reply:
Alright